spec/models/export/anything_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+RSpec.describe Export::Anything do
```

Comment:
I'd love to have a chat about the specs for this class. I found it very hard to follow, I think mostly because so much of it is relying on stubs and mocks. it also makes me wonder how much it is coupled to the underlying implementation, and whether that makes them particularly brittle. This may be down to experience and personal preference, but I always feel nervous when I see a spec file were all the actors have been stubbed/mocked. As a hypothetical, how much less confidence would we have in the test suite's coverage of the codebase if this spec file didn't exist at all? just wondering because we have the task specs covering each of the concrete examples.

Reply:
I'm going to either make this go away as a "file deleted, these were design-time specs" commit, or try and replace the stubs with `FactoryBot.create`  a bit, because as you point out, stubbing `:each` is a little bit too far here, and there's a block of four stubs that I have to take some time to parse...

Reply:
Given that I actually used the spec to update `Export::Anything` to tell us about the class that's missing when there are no records, it's probably useful, so I've de-`double`d it. It comes at the expense of a little factory complexity but I hope it's a bit less of a slog!